tests/builder/designspace_gen_test.py

```diff
@@ -164,8 +164,8 @@ def test_designspace_generation_brace_layers(datadir, filename, ufo_module):
         (a.name, a.minimum, a.default, a.maximum, a.map) for a in designspace.axes
     ]
     assert axes_order == [
-        ("Width", 75, 100, 100, [(75, 50.0), (100, 100.0)]),
-        ("Weight", 100, 100, 700, [(100, 100.0), (700, 1000.0)]),
+        ("Width", 75, 100, 100, [(75, 50), (87.5, 75), (100, 100)]),
+        ("Weight", 100, 100, 700, [(100, 100), (400, 500), (700, 1000)]),
```

Comment:
I presume these additional mappings coming from intermediate instances. Do they contain an explicit Axis Location parameter? If not, aren't these redundant? It looks like they are exactly the same value as it would be linearly interpolated between the previous and next mappings so they have no effect. I don't think varLib will prune those when making avar (maybe it could/should).

Reply:
The width axis mapping is redundant, the weight axis one not.

Regular has weight class 400, so IIUC in absence of any Axis Location parameters (on masters and instances), glyphsLib would deduce the mapped value from the weight class. Ignoring the weight class would put Regular at 366.6667.

Using the weight class to map Regular seems more correct to me.

But comparing fonts generated by Glyphs and glyphsLib, it seems that Glyphs doesn't bother mapping instances either in this constellation.

Reply:
>  it seems that Glyphs doesn't bother mapping instances either

which would be an argument for _not_ doing it here.. unless an explicit Axis Location custom parameter was used on the instance (which is what this PR presumably was about)?

Reply:
Yep. I'll change it…